front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ProjectsWidget/ProjectsCard/ProjectRow.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import {
+  Box,
+  Title,
+  Image,
+  Text,
+  stylingConsts,
+  Icon,
+  colors,
+} from '@citizenlab/cl2-component-library';
+import moment from 'moment';
+
+import { Period } from 'api/graph_data_units/responseTypes/ProjectsWidget';
+import { IProjectImageData } from 'api/project_images/types';
+import { IProjectData } from 'api/projects/types';
+
+import useLocalize from 'hooks/useLocalize';
+
+import { MessageDescriptor, useIntl } from 'utils/cl-intl';
+import { toFullMonth } from 'utils/dateUtils';
+
+import messages from '../messages';
+
+import { deriveProjectStatus, ProjectStatus } from './utils';
+
+interface Props {
+  project: IProjectData;
+  projectImage?: IProjectImageData;
+  period: Period;
+  participants: number;
+}
+
+const MESSAGES: Record<ProjectStatus, MessageDescriptor> = {
+  finished: messages.finished,
+  active: messages.active,
+  stale: messages.stale,
+  planned: messages.planned,
+};
+
+const LABEL_STYLE: Record<
+  ProjectStatus,
+  { background: string; color: string }
+> = {
+  finished: {
+    background: colors.teal100,
+    color: colors.teal500,
+  },
+  active: {
+    background: colors.green100,
+    color: colors.green500,
+  },
+  stale: {
+    background: colors.grey100,
+    color: colors.coolGrey600,
+  },
+  planned: {
+    background: colors.orange100,
+    color: colors.orange500,
+  },
+};
+
+const ProjectRow = ({ project, projectImage, period, participants }: Props) => {
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+
+  const startMonth = toFullMonth(period.start_at, 'month');
+  const endMonth = period.end_at
+    ? toFullMonth(period.end_at, 'month')
+    : formatMessage(messages.noEndDate);
+
+  const { title_multiloc } = project.attributes;
+
+  const status = deriveProjectStatus(period, moment());
+  const labelStyle = LABEL_STYLE[status];
+
+  return (
+    <Box display="flex" flexDirection="row" pb="16px">
+      {projectImage?.attributes.versions.large && (
+        <Image
+          src={projectImage?.attributes.versions.large}
+          alt={''}
```

Comment:
In this case, I think it's not strictly necessary because the image is there just to make things look nice. It doesn't really have any meaning. The one on the `ProjectCard` also doesn't have an alt description. It might even be annoying if you're going through this list of projects to have something like "This is an image representing the project" read out to you for each project